api/debug/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+
+  api:
+    image: opensdsio/multi-cloud-api-debug
+    volumes:
+      - /etc/ssl/certs:/etc/ssl/certs
+      - /etc/localtime:/etc/localtime
+      - .:/opt/go/src/github.com/opensds/multi-cloud
+    ports:
+      - "8089:8089"
+      - "2345:2345"
+    environment:
+      - MICRO_SERVER_ADDRESS=:8089
+      - MICRO_REGISTRY=mdns
+      - OS_AUTH_AUTHSTRATEGY=keystone
+      - OS_AUTH_URL=http://192.168.2.24/identity
```

Comment:
Please mention this IP needs to be changed to the user host ip, in the documentation.

Reply:
ok